fabric-data-attachment-api-v1/src/testmod/java/net/fabricmc/fabric/test/attachment/WheelInfo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.test.attachment;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.util.dynamic.Codecs;
+
+import net.fabricmc.fabric.api.attachment.v1.AttachmentRegistry;
+import net.fabricmc.fabric.api.attachment.v1.AttachmentType;
+
+public record WheelInfo(float wheelDiameter, float tireDiameter, float tireThickness) {
```

Comment:
This class is never loaded, so we never actually run this test code. I would suggest replacing this with a unit test in CommonAttachmentTests if possible.

Reply:
This was mostly included to show the API usage, rather than as an actual test, but I have moved it to the test package so it does get registered. I've also updated the existing tests to use the new API.